forge-gui/res/cardsfolder/upcoming/stormkeld_curator_giant_secrets.txt

```diff
@@ -0,0 +1,22 @@
+Name:Stormkeld Curator
+ManaCost:4 W
+Types:Creature Giant
+PT:6/6
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigAurify | TriggerDescription$ When CARDNAME enters the battlefield, you may put any number of Aura cards from your graveyard and/or hand onto the battlefield attached to it.
+SVar:TrigAurify:DB$ ChangeZone | Origin$ Hand | Destination$ Battlefield | ChangeType$ Aura.CanEnchantSource+YouOwn | AttachedTo$ Self | ChangeNum$ HandX | Optional$ True | Hidden$ True | SubAbility$ GraveAuras
```

Comment:
any reason it can't be combined into a single ChangeZone?

Reply:
the Auras probably needs to enter at the same time

Reply:
Yeah I copied this part almost entirely from Bruna, Light of Alabaster (mistakenly) thinking they were the same rules wise 😱